### PR TITLE
Feature/share link

### PR DESCRIPTION
**Pull request by jackfrancismurphy**
Share link added 
- Button appears when winning conditions met
- Button to be made pretty under broad CSS renovation

### CODE REVIEW

File: newdle_frontend.js [hunk 1]
```diff
@@ -63,25 +72,56 @@ function checkGuess() {
...
+    if (time_taken >= 6 && time_taken <8){
+        mins_stamp = "<8"
+    }
+    if (time_taken >= 8 && time_taken <10){
+        mins_stamp = "<10"
+    }
+    if (time_taken >10){
+        mins_stamp = ">10"
+    }
+    link = `🟩📰⌛${mins_stamp}\n "http://localhost:8000/"`
```
Comment (JackDanielHarding) [inline newdle_frontend.js:122]: You need to remove the quotations around the link.

**Review by JackDanielHarding — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -21,9 +21,12 @@ <h2 id="presented_headline"> </h2>
         <input type="text" id="guess_box" name="guess"><br><br>
         <p id= "cross"></p>
         </div>
+        <button type="button" id="submit_button" onclick="onSubmit()">Guess</button> 
+        <button type="button" id="share_button" onclick="shareLink()">Share</button> 
         <button type="button" id="submit_button" onclick="onSubmit()"></button>
```
Comment (JackDanielHarding) [inline index.html:26]: Why do you have a second submit button?

**Review by JackDanielHarding — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -28,7 +28,9 @@ <h2 id="presented_headline"> </h2>
         <input type="text" id="guess_box" name="guess"><br><br>
         <p id= "cross"></p>
         </div>
-        <button type="button" id="submit_button" onclick="onSubmit()"></button>
```
Comment (JackDanielHarding) [inline index.html:31]: You've notw removed the submit button completely

**Review by JackDanielHarding — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (jackfrancismurphy) [reply index.html:31]: remind me how we move forward with this ticket? I now have a new issue with the 'submit with enter' feature. But I need to know how I resolve this before I can carry on

**Review by JackDanielHarding — APPROVED** [review]

**Pull request merged**